runtime/entrypoint.sh

```diff
@@ -33,9 +33,12 @@ exit_code=0
     if [ -f "main.py" ]
     then
         echo "Generating descriptors on a subset of query videos..."
+        echo "Started at $(date +%s)"
 
         conda run --no-capture-output -n condaenv python main.py
 
+        echo "Finished at $(date +%s)
```

Comment:
Why `%s`? I think this should be human readable. I propose doing an ISO 8601 timestamp:

```bash
❯ date -u +"%Y-%m-%dT%H:%M:%SZ"
2023-03-08T16:33:26Z
```

Reply:
I would do both then in that case. 

```
❯ echo "$(date -u +'%Y-%m-%dT%H:%M:%SZ') ($(date +%s))"
2023-03-08T16:37:53Z (1678293473)
```